test/test_base.py

```diff
@@ -259,8 +274,13 @@ def test_type_cast(
             not torch.cuda.is_available() or torch.cuda.get_device_capability() < (9, 0)
         )
 
+
+        fp8_dtypes = (
+            FP8Dtypes()
+        )
+
         m = nn.Linear(32, 16, device="cuda", dtype=linear_dtype)
-        m = get_float8_linear(linear_type, m, emulate, False)
+        m = get_float8_linear(linear_type, m, emulate, False, fp8_dtypes)
```

Comment:
you can enable emulation here if your hardware doesn't support the dtype under test